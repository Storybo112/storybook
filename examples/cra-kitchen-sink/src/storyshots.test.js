describe('Storyshots', () => {
  xit('should run snapshot tests, but we\'ve disabled this temporarily', () => {});
});

// NOTE: this file should contain a snapshot test, but it is temporarily disabled.
//
// From @tmeasday: "Both Lerna/npm5 and Jest are incompatible, so we cannot run
// Jest tests right now, unless we go to great lengths (see `test-cra`'s build process)."
//
// A succinct repro here: https://github.com/tmeasday/preserve-symlinks-test
//
// Once this difference is resolved, we should uncomment the following code:
//
// import initStoryshots, { snapshotWithOptions } from '@storybook/addon-storyshots';
// import path from 'path';
//
// function createNodeMock(element) {
//   if (element.type === 'div') {
//     return { scrollWidth: 123 };
//   }
//   return null;
// }
//
// initStoryshots({
//   framework: 'react',
//   configPath: path.join(__dirname, '..', '.storybook'),
//   test: snapshotWithOptions({
//     createNodeMock,
//   }),
// });
