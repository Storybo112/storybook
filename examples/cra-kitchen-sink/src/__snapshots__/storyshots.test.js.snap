// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Addons composition with some emoji 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots Addons composition with text 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  Hello Button
</button>
`;

exports[`Storyshots App full app 1`] = `
<div
  className="App"
>
  <div
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="file-stub"
    />
    <h2>
      Welcome to React
    </h2>
  </div>
  <p
    className="App-intro"
  >
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;

exports[`Storyshots Button with knobs 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#ffff00",
      "border": "3px solid #ff00ff",
      "padding": "10px",
    }
  }
>
  <p>
    My name is Storyteller, I'm 70 years old, and my favorite fruit is apple.
  </p>
  <p>
    My birthday is: 
    January 20, 2017
  </p>
  <p>
    My wallet contains: $
    12.50
  </p>
  <p>
    In my backpack, I have:
  </p>
  <ul>
    <li>
      Laptop
    </li>
    <li>
      Book
    </li>
    <li>
      Whiskey
    </li>
  </ul>
  <p>
    Nice to meet you!
  </p>
</div>
`;

exports[`Storyshots Button with notes 1`] = `
<button
  className="css-1yjiefr"
>
  Check my notes in the notes panel
</button>
`;

exports[`Storyshots Button with setInfoOptions 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  Button with Info
</button>
`;

exports[`Storyshots Button with some emoji 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots Button with text 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  Hello Button
</button>
`;

exports[`Storyshots Cells.Molecules.Atoms with some emoji2 1`] = `
<button
  className="css-1yjiefr"
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots Cells.Molecules.Atoms with text2 1`] = `
<button
  className="css-1yjiefr"
>
  Hello Button
</button>
`;

exports[`Storyshots Cells/Molecules with some emoji 1`] = `
<button
  className="css-1yjiefr"
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots Cells/Molecules with text 1`] = `
<button
  className="css-1yjiefr"
>
  Hello Button
</button>
`;

exports[`Storyshots Cells/Molecules.Atoms/simple with some emoji 1`] = `
<button
  className="css-1yjiefr"
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots Cells/Molecules.Atoms/simple with text 1`] = `
<button
  className="css-1yjiefr"
>
  Hello Button
</button>
`;

exports[`Storyshots Cells/Molecules/Atoms.more with some emoji2 1`] = `
<button
  className="css-1yjiefr"
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots Cells/Molecules/Atoms.more with text2 1`] = `
<button
  className="css-1yjiefr"
>
  Hello Button
</button>
`;

exports[`Storyshots Info Addon DocgenButton 1`] = `
<button
  disabled={false}
  onClick={[Function]}
>
  Docgen Button
</button>
`;

exports[`Storyshots Info Addon FlowTypeButton 1`] = `
<button
  disabled={false}
  onClick={[Function]}
>
  Flow Typed Button
</button>
`;

exports[`Storyshots Info Addon Info default 1`] = `
<div
  title="the best container ever"
>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  Info default
  "
  
</div>
`;

exports[`Storyshots Info Addon Info with options 1`] = `
<div
  title="the best container ever"
>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  Info with options
  " this button has very-very-very-very looooooong title
  
</div>
`;

exports[`Storyshots Info Addon JSX as an argument 1`] = `
<div
  title="the best container ever"
>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  JSX as an argument
  "
  
</div>
`;

exports[`Storyshots Info Addon Only component source 1`] = `
<div
  title="the best container ever"
>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  Only component source
  "
  
</div>
`;

exports[`Storyshots Info Addon Only propTypes and header 1`] = `
<div>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  Only propTypes and header
  "
</div>
`;

exports[`Storyshots Info Addon Only summary with markdown 1`] = `
<div
  title="the best container ever"
>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  Only summary with markdown
  "
  
</div>
`;

exports[`Storyshots Info Addon React Element in summary 1`] = `
<div
  title="the best container ever"
>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  React Element in summary
  "
  
</div>
`;

exports[`Storyshots Info Addon With custom components 1`] = `
<div
  title="the best container ever"
>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  With custom components
  "
  
</div>
`;

exports[`Storyshots Info Addon addons composition 1`] = `
<div>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  addons composition
  "
</div>
`;

exports[`Storyshots Info Addon decoratorInfo 1`] = `
<div>
  <div
    style={
      Object {
        "display": false,
      }
    }
  >
    <div
      style={
        Object {
          "position": "relative",
          "zIndex": 0,
        }
      }
    >
      <div
        title="the best container ever"
      >
        click the 
        <span
          style={
            Object {
              "background": "rgb(34, 136, 204)",
              "borderRadius": "0px 0px 0px 5px",
              "color": "rgb(255, 255, 255)",
              "fontFamily": "sans-serif",
              "fontSize": 12,
              "margin": 10,
              "padding": "5px 15px",
              "textDecoration": "none",
            }
          }
        >
           
          Show Info
           
        </span>
         label in top right for info about "
        decoratorInfo
        "
        
      </div>
    </div>
    <a
      onClick={[Function]}
      role="button"
      style={
        Object {
          "background": "#28c",
          "borderRadius": "0 0 0 5px",
          "color": "#fff",
          "cursor": "pointer",
          "display": false,
          "fontFamily": "sans-serif",
          "fontSize": "12px",
          "padding": "5px 15px",
          "position": "fixed",
          "right": 0,
          "textDecoration": "none",
          "top": 0,
        }
      }
      tabIndex="0"
    >
      Show Info
    </a>
  </div>
  <div
    style={
      Object {
        "bottom": 0,
        "display": "none",
        "left": 0,
        "overflow": "auto",
        "position": "relative",
        "right": 0,
        "top": 0,
        "zIndex": 99999,
      }
    }
  >
    <a
      onClick={[Function]}
      role="button"
      style={
        Object {
          "background": "#28c",
          "borderRadius": "0 0 0 5px",
          "color": "#fff",
          "cursor": "pointer",
          "display": false,
          "fontFamily": "sans-serif",
          "fontSize": "12px",
          "padding": "5px 15px",
          "position": "fixed",
          "right": 0,
          "textDecoration": "none",
          "top": 0,
        }
      }
      tabIndex="0"
    >
      √ó
    </a>
    <div
      style={undefined}
    >
      <div
        style={
          Object {
            "WebkitFontSmoothing": "antialiased",
            "backgroundColor": "#fff",
            "border": "1px solid #eee",
            "borderRadius": "2px",
            "boxShadow": "0px 2px 3px rgba(0, 0, 0, 0.05)",
            "color": "#444",
            "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
            "fontSize": "15px",
            "fontWeight": 300,
            "lineHeight": 1.45,
            "padding": "10px 20px 20px",
          }
        }
      >
        <div
          style={
            Object {
              "borderBottom": "1px solid #eee",
              "marginBottom": 6,
              "paddingTop": 6,
            }
          }
        >
          <h1
            style={
              Object {
                "fontSize": "28px",
                "margin": 0,
                "padding": 0,
              }
            }
          >
            Info Addon
          </h1>
          <h2
            style={
              Object {
                "fontSize": "18px",
                "fontWeight": 400,
                "margin": "0 0 10px 0",
                "padding": 0,
              }
            }
          >
            decoratorInfo
          </h2>
        </div>
        <div
          style={
            Object {
              "borderBottom": "1px solid #eee",
              "marginBottom": 0,
            }
          }
        >
          <div>
            <p
              style={
                Object {
                  "backgroundColor": "hsl(0, 0%, 100%)",
                  "color": "hsl(218, 54%, 20%)",
                  "fontFamily": "-apple-system,BlinkMacSystemFont,\\"Segoe UI\\",Roboto,Oxygen,Ubuntu,\\"Fira Sans\\",\\"Droid Sans\\",\\"Helvetica Neue\\",sans-serif",
                  "fontSize": 14,
                }
              }
            >
              Use the 
              <a
                href="https://github.com/storybooks/storybook/tree/master/addons/info"
                style={
                  Object {
                    "color": "hsl(216, 100%, 40%)",
                    "textDecoration": "none",
                  }
                }
              >
                info addon
              </a>
               with its new painless API.
            </p>
          </div>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "color": "rgba(0,0,0,0.5)",
                "fontSize": "18px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Story Source
          </h1>
          <pre
            className="fenced-code"
            style={
              Object {
                "border": "1px hsl(0, 0%, 80%) solid",
                "borderRadius": 3,
                "fontSize": 13,
                "padding": "1px 3px",
              }
            }
          >
            <code
              className="hljs"
            >
              <div>
                <div
                  style={
                    Object {
                      "paddingLeft": 18,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    &lt;
                    Container
                  </span>
                  <span />
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    &gt;
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 33,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    click the 
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 33,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    &lt;
                    InfoButton
                  </span>
                  <span />
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    /&gt;
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 33,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                     label in top right for info about "
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 33,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    decoratorInfo
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 33,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    "
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 18,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "hsl(0, 0%, 10%)",
                      }
                    }
                  >
                    &lt;/
                    Container
                    &gt;
                  </span>
                </div>
              </div>
            </code>
          </pre>
        </div>
        <div>
          <h1
            style={
              Object {
                "borderBottom": "1px solid #EEE",
                "color": "rgba(0,0,0,0.5)",
                "fontSize": "18px",
                "margin": "20px 0 0 0",
                "padding": "0 0 5px 0",
              }
            }
          >
            Prop Types
          </h1>
          <div>
            <h2
              style={
                Object {
                  "fontSize": "14px",
                  "margin": "20px 0 10px 0",
                }
              }
            >
              <span
                style={
                  Object {
                    "backgroundColor": "hsl(0, 0%, 92%)",
                    "border": "solid 1px hsl(0, 0%, 75%)",
                    "borderRadius": 4,
                    "minWidth": 300,
                    "padding": "6px 20px",
                  }
                }
              >
                &lt;
                Container
                 /&gt;
              </span>
            </h2>
            <table
              style={
                Object {
                  "backgroundColor": "hsl(0, 0%, 98%)",
                  "border": "1px hsl(0, 0%, 90%) solid",
                  "borderCollapse": "separate",
                  "fontSize": 13,
                  "width": "100%",
                }
              }
            >
              <thead
                style={
                  Object {
                    "textAlign": "left",
                  }
                }
              >
                <tr>
                  <th>
                    property
                  </th>
                  <th>
                    propType
                  </th>
                  <th>
                    required
                  </th>
                  <th>
                    default
                  </th>
                  <th>
                    description
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    children
                  </td>
                  <td>
                    node
                  </td>
                  <td>
                    yes
                  </td>
                  <td>
                    -
                  </td>
                  <td />
                </tr>
                <tr>
                  <td>
                    isAmazing
                  </td>
                  <td>
                    bool
                  </td>
                  <td>
                    no
                  </td>
                  <td>
                    <span>
                      <span
                        style={
                          Object {
                            "color": "#a11",
                          }
                        }
                      >
                        false
                      </span>
                    </span>
                  </td>
                  <td />
                </tr>
                <tr>
                  <td>
                    age
                  </td>
                  <td>
                    number
                  </td>
                  <td>
                    no
                  </td>
                  <td>
                    <span>
                      <span
                        style={
                          Object {
                            "color": "#a11",
                          }
                        }
                      >
                        0
                      </span>
                    </span>
                  </td>
                  <td />
                </tr>
                <tr>
                  <td>
                    title
                  </td>
                  <td>
                    string
                  </td>
                  <td>
                    no
                  </td>
                  <td>
                    <span
                      style={
                        Object {
                          "color": "#22a",
                          "wordBreak": "break-word",
                        }
                      }
                    >
                      "
                      the best container ever
                      "
                    </span>
                  </td>
                  <td />
                </tr>
              </tbody>
            </table>
          </div>
          <div>
            <h2
              style={
                Object {
                  "fontSize": "14px",
                  "margin": "20px 0 10px 0",
                }
              }
            >
              <span
                style={
                  Object {
                    "backgroundColor": "hsl(0, 0%, 92%)",
                    "border": "solid 1px hsl(0, 0%, 75%)",
                    "borderRadius": 4,
                    "minWidth": 300,
                    "padding": "6px 20px",
                  }
                }
              >
                &lt;
                InfoButton
                 /&gt;
              </span>
            </h2>
            <small>
              No propTypes defined!
            </small>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Info Addon deprecated addWithInfo 1`] = `
<div>
  click the 
  <span
    style={
      Object {
        "background": "rgb(34, 136, 204)",
        "borderRadius": "0px 0px 0px 5px",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "sans-serif",
        "fontSize": 12,
        "margin": 10,
        "padding": "5px 15px",
        "textDecoration": "none",
      }
    }
  >
     
    Show Info
     
  </span>
   label in top right for info about "
  deprecated addWithInfo
  "
</div>
`;

exports[`Storyshots Navigation Menu link 1`] = `
<div
  className="css-t9df35"
>
  <a
    className="css-1enjukp"
    href="/"
    onClick={[Function]}
  >
    Menu link item
  </a>
</div>
`;

exports[`Storyshots Navigation Routed link 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  Try clicking with different mouse buttons and modifier keys (shift/ctrl/alt/cmd)
</a>
`;

exports[`Storyshots Some really long story kind description with text 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "display": "flex",
      "justifyContent": "center",
      "left": 0,
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "margin": "auto",
      }
    }
  >
    <button
      className="css-1yjiefr"
      onClick={[Function]}
    >
      Hello Button
    </button>
  </div>
</div>
`;

exports[`Storyshots Welcome to Storybook 1`] = `
<article
  className="css-1fqbdip"
>
  <h1
    className="css-nil"
  >
    Welcome to storybook
  </h1>
  <p>
    This is a UI component dev environment for your app.
  </p>
  <p>
    We've added some basic stories inside the
     
    <code
      className="css-mteq83"
    >
      src/stories
    </code>
     
    directory.
    <br />
    A story is a single state of one or more UI components. You can have as many stories as you want.
    <br />
    (Basically a story is like a visual test case.)
  </p>
  <p>
    See these sample
     
    <a
      className="css-ca0824"
      onClick={[Function]}
      role="button"
      tabIndex="0"
    >
      stories
    </a>
     
    for a component called
     
    <code
      className="css-mteq83"
    >
      Button
    </code>
    .
  </p>
  <p>
    Just like that, you can add your own components as stories.
    <br />
    You can also edit those components and see changes right away.
    <br />
    (Try editing the 
    <code
      className="css-mteq83"
    >
      Button
    </code>
     stories located at 
    <code
      className="css-mteq83"
    >
      src/stories/index.js
    </code>
    .)
  </p>
  <p>
    Usually we create stories with smaller UI components in the app.
    <br />
    Have a look at the
     
    <a
      className="css-ca0824"
      href="https://storybook.js.org/basics/writing-stories"
      rel="noopener noreferrer"
      target="_blank"
    >
      Writing Stories
    </a>
     
    section in our documentation.
  </p>
  <p
    className="css-bwdon3"
  >
    <b>
      NOTE:
    </b>
    <br />
    Have a look at the
     
    <code
      className="css-mteq83"
    >
      .storybook/webpack.config.js
    </code>
     
    to add webpack loaders and plugins you are using in this project.
  </p>
</article>
`;

exports[`Storyshots WithEvents Logger 1`] = `
<div
  style={
    Object {
      "color": "rgb(51, 51, 51)",
      "fontFamily": "
          -apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", \\"Roboto\\",
          \\"Segoe UI\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", sans-serif
        ",
      "padding": 20,
    }
  }
>
  <h1>
    Logger
  </h1>
  <dl />
</div>
`;

exports[`Storyshots component.Button first 1`] = `
<button>
  first button
</button>
`;

exports[`Storyshots component.Button second 1`] = `
<button>
  first second
</button>
`;

exports[`Storyshots component.base.Link first 1`] = `
<a>
  first link
</a>
`;

exports[`Storyshots component.base.Link second 1`] = `
<a>
  second link
</a>
`;

exports[`Storyshots component.base.Span first 1`] = `
<span>
  first span
</span>
`;

exports[`Storyshots component.base.Span second 1`] = `
<span>
  second span
</span>
`;

exports[`Storyshots component.common.Div first 1`] = `
<div>
  first div
</div>
`;

exports[`Storyshots component.common.Div second 1`] = `
<div>
  second div
</div>
`;

exports[`Storyshots component.common.Table first 1`] = `
<table>
  <tr>
    <td>
      first table
    </td>
  </tr>
</table>
`;

exports[`Storyshots component.common.Table second 1`] = `
<table>
  <tr>
    <td>
      first table
    </td>
  </tr>
</table>
`;

exports[`Storyshots withNotes with a button and some emoji 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots withNotes with old API 1`] = `
<button
  className="css-1yjiefr"
  onClick={[Function]}
>
  üòÄ üòé üëç üíØ
</button>
`;

exports[`Storyshots withNotes with some emoji 1`] = `
<p>
  ü§îüò≥üòØüòÆ
</p>
`;

exports[`Storyshots withNotes with some text 1`] = `
<div>
  Hello guys
</div>
`;
